grafana/resource_alert_notification.go

```diff
@@ -0,0 +1,128 @@
+package grafana
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	gapi "github.com/apparentlymart/go-grafana-api"
+)
+
+func ResourceAlertNotification() *schema.Resource {
+	return &schema.Resource{
+		Create: CreateAlertNotification,
+		Update: UpdateAlertNotification,
+		Delete: DeleteAlertNotification,
+		Read:   ReadAlertNotification,
+
+		Schema: map[string]*schema.Schema{
+			"id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"type": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"is_default": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"settings": {
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func CreateAlertNotification(a *schema.ResourceData, meta interface{}) error {
```

Comment:
Minor Nit: Conventionally we use `d` for `*schema.ResourceData` inside of the CRUD functions.